srv/src/test/java/my/bookshop/AdminServiceTest.java

```diff
@@ -31,6 +34,9 @@ public class AdminServiceTest {
 	@Resource(name = AdminService_.CDS_NAME)
 	private DraftService adminService;
 
+	@Resource(name = AdminService_.CDS_NAME)
+	private ApplicationService activeService;
```

Comment:
Why not simply use the existing `adminService` instance?

Reply:
I wanted to bypass the drafts. Changed to DraftService.run().